eventOrganizer/src/main/java/com/event/organizer/api/controller/EventController.java

```diff
@@ -51,6 +52,11 @@ public String deleteEvent(@RequestBody EventRequestDto eventRequestDto) throws E
         return "deleted";
     }
 
+    @PostMapping
+    public void AddComment(Comment comment, Event event) throws EventOrganizerException {

```

Comment:
Change this to take (string comment, int eventId)  instead as the UI already implements it this way